src/app/mod.rs

```diff
@@ -26,8 +26,17 @@ fn print_options(config_filename: &str, tokens: &[String], last_token: &str) ->
     let options_finder = options_finder::OptionsFinder::new(result);
     let opts = options_finder.options(last_token)?;
 
+    let print_descriptions = std::env::var("TABRY_PRINT_DESCRIPTIONS").is_ok();
```

Comment:
Is there a reason you chose a env var over a flag? I think I would prefer a flag, although I can't explain why, it just somehow seems more suitable, so I could be convinced otherwise.

Reply:
That was just how I implemented it in the ruby version, I can switch to a flag, no problem!